<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1" rowspan="2">
       <widget class="ParachuteView" name="parachuteView" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="BinaryViewWidget" name="binaryView" native="true"/>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1" colspan="4">
         <widget class="QLineEdit" name="messageLineEdit"/>
        </item>
        <item row="3" column="2">
         <widget class="QComboBox" name="color1">
          <item>
           <property name="text">
            <string>Red</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Black</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blue</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Brown</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Enter a message:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QSlider" name="tracks_slider">
          <property name="value">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QSlider" name="sectorSlider">
          <property name="value">
           <number>21</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QComboBox" name="color0">
          <item>
           <property name="text">
            <string>White</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Yellow</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pink</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Orange</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Choose number tracks</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>1:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Colors:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QSpinBox" name="nbTrack">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>0:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Choose number sector</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QSpinBox" name="nbSector">
          <property name="minimum">
           <number>3</number>
          </property>
          <property name="value">
           <number>21</number>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QSpinBox" name="shiftVal">
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="shift">
          <property name="text">
           <string>Enter code of @ :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save_as"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_About_us">
    <property name="title">
     <string>&amp;About us</string>
    </property>
    <addaction name="action_Description"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_About_us"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Save_as">
   <property name="text">
    <string>Sa&amp;ve as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="iconText">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Description">
   <property name="text">
    <string>&amp;Description</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BinaryViewWidget</class>
   <extends>QWidget</extends>
   <header>BinaryViewWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ParachuteView</class>
   <extends>QWidget</extends>
   <header>ParachuteView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
